### PR TITLE
feat: finish redux Logic and UI for Guest Screen

### PR DESCRIPTION
**Pull request by SpQuyt**

### CODE REVIEW

File: src/components/Home/EventDetail/Guest/Unaccepted/index.js [hunk 1]
```diff
@@ -1,25 +1,56 @@
...
+        </View>
+        <FlatList
+          style={styles.listContainer}
+          data={unacceptedGuestsList}
+          extraData={unacceptedGuestsList}
+          keyExtractor={(item) => item._id}
+          renderItem={({ item }) => (
+            <View style={styles.listItem}>
+              <Text style={styles.text}>{item.email}</Text>
+              <TouchableOpacity onPress={() => {}} style={styles.button}>
```
Comment (namdaoduy) [inline src/components/Home/EventDetail/Guest/Unaccepted/index.js]: Logic will be implement in another PR?

Comment (SpQuyt) [reply src/components/Home/EventDetail/Guest/Unaccepted/index.js]: I have added Logic

File: .vscode/settings.json [hunk 2]
```diff
@@ -3,5 +3,8 @@
   "files.trimTrailingWhitespace": true,
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
-  "eslint.autoFixOnSave": true
+  "eslint.autoFixOnSave": true,
```
Comment (namdaoduy) [inline .vscode/settings.json]: Remove `"eslint.autoFixOnSave": true,`

File: src/configs/index.js [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
 
 import baseConfig from './base';
 import localConfig from './local';
 import devConfig from './dev';
 import prodConfig from './prod';
 
-const env = process.env.REACT_APP_ENV;
+const env = REACT_APP_ENV;
 let envConfig = {};
+console.log(REACT_APP_ENV);
```
Comment (namdaoduy) [inline src/configs/index.js]: Remove after debugging

File: .vscode/settings.json (hunk 2 above)
Comment (SpQuyt) [reply .vscode/settings.json]: ok

File: src/configs/index.js (hunk 3 above)
Comment (SpQuyt) [reply src/configs/index.js]: ok
